tools/pd-ctl/pdctl/command/region_command.go

```diff
@@ -65,21 +67,37 @@ func NewRegionCommand() *cobra.Command {
 	r.AddCommand(NewRegionsByKeysCommand())
 	r.AddCommand(NewRangesWithRangeHolesCommand())
 
-	topRead := &cobra.Command{
+	topReadFlow := &cobra.Command{
 		Use:   `topread <limit> [--jq="<query string>"]`,
 		Short: "show regions with top read flow",
 		Run:   showRegionsTopCommand(regionsReadFlowPrefix),
 	}
-	topRead.Flags().String("jq", "", "jq query")
-	r.AddCommand(topRead)
+	topReadFlow.Flags().String("jq", "", "jq query")
+	r.AddCommand(topReadFlow)
 
-	topWrite := &cobra.Command{
+	topWriteFlow := &cobra.Command{
 		Use:   `topwrite <limit> [--jq="<query string>"]`,
 		Short: "show regions with top write flow",
 		Run:   showRegionsTopCommand(regionsWriteFlowPrefix),
 	}
-	topWrite.Flags().String("jq", "", "jq query")
-	r.AddCommand(topWrite)
+	topWriteFlow.Flags().String("jq", "", "jq query")
+	r.AddCommand(topWriteFlow)
+
+	topReadQuery := &cobra.Command{
+		Use:   `topread-query <limit> [--jq="<query string>"]`,
```

Comment:
How about not using `-`? BTW, do we need the key dimension?

Reply:
I'm confused by this command name. It made me expect an actual query, not information about regions.

Reply:
How unified to  `topreadflow` and `topreadquery`. or `topread flow`, `topread query` as a parameter after topread.

Reply:
LGTM